mlflow/ml-package-versions.yml

```diff
@@ -481,7 +481,16 @@ transformers:
     maximum: "4.28.1"
     requirements:
       ">= 0.0.0":
-        ["datasets", "huggingface_hub", "torch", "torchvision", "tensorflow", "accelerate"]
+        [
+          "datasets",
+          "huggingface_hub",
+          "torch",
+          "torchvision",
+          "tensorflow",
+          "accelerate",
+          "librosa",
+          "ffmpeg",
```

Comment:
If possible, can we add some inline comments explaining which libraries are required for which functionalities?